### PR TITLE
Maintenance Grant Application Template

### PR DESCRIPTION
**Pull request by enfipy**
### Maintenance Grant Application Template

This is an example of the Maintenance Grant Application. Feel free to update or add any information to this template file.

Also, maybe we want to change the Development Roadmap example. I think it's a subject of change for most Projects that will apply for Maintenance Grant. Will highly appreciate any feedback.

### CODE REVIEW

**Review by alxs — CHANGES_REQUESTED** [review]
Thanks @enfipy, this is amazing.

My only suggestions would be that levels shouldn't apply to maintenance grants and also the development roadmap should probably be worded/structured a bit differently as this type of grant is meant to cover project maintenance in the stricter meaning (see #671, and also apologies as this was really not clear before).

Feel free to let us take care of the changes, I'm OOO until next week but maybe someone else will pick it up.

**Review by Noc2 — CHANGES_REQUESTED** [review]
Just an additional note here, also once @alxs is back. He can work on it: The development roadmap should mention that all code is published under an open source license during the maintenance period.

**Comment by enfipy** [comment]
@alxs and @Noc2, thanks for clarifying these things! As you requested, I changed some parts and if you want to change something else - let me know :)

File: maintenance/application-template.md [hunk 1]
```diff
@@ -85,26 +84,31 @@ If you've already started implementing your project or it is part of a larger re
...
+This section should specify what kind of tasks and responsibilities the maintainer team will cover during future development. If you have already outlined a list of issues/bugs or pull requests that need further development - you can specify them here to provide more context on what tasks you will close.
 
-Make sure the current project owners are willing to review/accept your contributions (a note here: if you're fully taking over the project, it will make more sense for the current owners to transfer the repository to your organization. If you can't get in touch with them, you may, of course, work on a fork).
+Also, make sure the current project owners are willing to review/accept your contributions (a note here: if you're fully taking over the project, it will make more sense for the current owners to transfer the repository to your organization. If you can't get in touch with them, you may, of course, work on a fork).
 
-Below we provide an **example roadmap**.
+Below we provide an **maintenance responsibilities**.
```
Comment (semuelle) [inline maintenance/application-template.md:93]:
```suggestion
Below we provide **example maintenance responsibilities**.
```

**Review by Noc2 — APPROVED** [review]
Looks good to me. Thanks for all the work!

**Review by semuelle — APPROVED** [review]

**Review by mmagician — APPROVED** [review]

**Pull request merged**